src/common/components/contactInfo/contactInfo.component.js

```diff
@@ -99,11 +99,14 @@ class Step1Controller {
           }
         }
 
-        const firstTimeLoading = !find(this.donorDetails.links, ['rel', 'donormatchesform'])
-        if (overrideDonorDetails && firstTimeLoading) {
-          this.donorDetails = assign(this.donorDetails, pick(overrideDonorDetails, [
-            'donor-type', 'name', 'organization-name', 'phone-number', 'spouse-name', 'mailingAddress', 'email'
-          ]))
+        if (overrideDonorDetails) {
+          const initialLoad = !this.$window.sessionStorage.getItem('initialLoad')
```

Comment:
The session storage variable seems a bit confusing here. If I were to read `initialLoad = true` I would think it has not loaded before. Perhaps something more like `initiallyLoaded` or `alreadyLoaded` or `initialLoadComplete` would be less confusing.

Reply:
That makes sense. I'll change it to `initialLoadComplete`. Thanks